bitnami/minio/templates/gateway/deployment.yaml

```diff
@@ -171,6 +175,18 @@ spec:
           emptyDir: {}
           {{- end }}
         {{- end }}
+        {{- if .Values.tls.enabled }}
+        - name: minio-certs
+          secret:
+            secretName: {{ include "minio.tlsSecretName" . }}
+            items:
+            - key: tls.crt
+              path: public.crt
+            - key: tls.key
+              path: private.key
+            - key: ca.crt
+              path: CAs/public.crt
```

Comment:
Why this path? Second question: should this block be optional?

Reply:
Also, the names are not optional.

According to MinIO documentation, the relative path is built using `--certs-dir` (https://docs.min.io/docs/how-to-secure-access-to-minio-server-with-tls.html)

So, renaming from Kubernetes/tls key structure to MinIO's mandatory name policy:
- `ca.crt -> {certs-dir}/CAs/public.crt`
- `tls.crt -> {certs-dir}/public.crt`
- `tls.key -> {certs-dir}/private.key`

Reply:
👍